src/lib/download.js

```diff
@@ -18,10 +18,20 @@ export default (filename) => {
       stream: process.stdout
     }, progress.Presets.rect)
 
+    if (attempts === 0) {
+      reject(new Error(clk.red(`Unable to download ${fullUrl}`)))
```

Comment:
use a better error message, e.g. failed to download file .... after ... attempts